src/cache/redis/RedisClient.js

```diff
@@ -0,0 +1,15 @@
+const redis = require("redis");
+const { promisifyRedisClient } = require("./utils");
+
+const makeNewRedisClient = (redisConfig) => redis.createClient(redisConfig);
```

Comment:
maybe rename to `createRedisClient`, to be consistent with the inner function call `redis.createClient` ? not critical though, uty